cluster_test.go

```diff
@@ -590,12 +596,11 @@ func TestClusterNodeWithoutData(t *testing.T) {
 
 	// Because it's behind, it's expected that the first node will flap when it
 	// proxies requests and gets v2 from peers.
-	// tc.sequinses[0].expectProgression(down, noVersion, v1, v3)
-	tc.sequinses[1].expectProgression(down, noVersion, v1, v2, v3)
-	tc.sequinses[2].expectProgression(down, noVersion, v1, v2, v3)
+	// tc.sequinses[0].expectProgression(v1, v3)
```

Comment:
is there a reason this is commented out?

Reply:
My PR didn't comment this out, it was already like that (0eda089523d171507373fc68cf8d2b9eeb801473). I think the comment above it is saying that this is what we would test, but we can't because it will be flappy (expected).